Makefile

```diff
@@ -274,6 +274,10 @@ test-plugin: build ${PROTOC_GEN_SWIFT}
 	done
 	diff -ru _test Reference
 
+# Test the SPM plugin.
+test-spm-plugin:
```

Comment:
I guess we can't add this to `check` since it would then run on the swift versions that don't support plugins.  Guess we just rely on CI to catch things and folks to remember to manually screen things.

Reply:
Yeah that was the idea. In a best world we would actually write these as normal tests. However, we need to solve the protoc problem in the ecosystem first.